JASP-Tests/R/tests/testthat/test-multinomialtest.R

```diff
@@ -42,3 +42,30 @@ test_that("Descriptives plot matches", {
   testPlot <- results[["state"]][["figures"]][[1]][["obj"]]
   expect_equal_plots(testPlot, "descriptives-1", dir="MultinomialTest")
 })
+
+test_that("Analysis handles errors - Negative Values", {
+  options <- jasptools::analysisOptions("MultinomialTest")
+  options$factor <- "facExperim"
+  options$counts <- "contNormal"
+  results <- jasptools::run("MultinomialTest", "test.csv", options)
+  errorMsg <- results[["results"]][["errorMessage"]]
+  expect_is(errorMsg, "character")
```

Comment:
The msg at `results[["results"]][["errorMessage"]]` is always a character, whether the error was handled properly (through `.hasErrors()`) or not caught at all. It's better to test that `results[["status"]]` is equal to `"validationError"`